projects/01-rubygems-cli/solution/lib/command_results/command_error_result.rb

```diff
@@ -0,0 +1,14 @@
+require "./lib/command_results/help_command_result"
+
+class CommandErrorResult < HelpCommandResult
+  attr_accessor :error_message
+
+  def initialize(error_message, exit_code = 1)
+    super(exit_code)
+    @error_message = error_message
+  end
+
+  def output
+    error_message + "\n" + @@help_message
```

Comment:
See https://github.com/stefanradu904/Cli_Program/pull/1#discussion_r836103147.

Regarding class variables (aka `@@help_message`):
![image](https://user-images.githubusercontent.com/24190790/160568033-cef3f9fb-2bf6-4c8c-87d8-71706a28b1fa.png)

If we'd use Rubocop (ruby's most popular linter), you'd receive [this warning](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ClassVars).

They're more to be seen _like_ constants, but instead of using a constant alternative, better use an actual constant:
```ruby
SOME_CONSTANT = 'hi mom'

class Abc
  def asd
    puts SOME_CONSTANT.uppercase
  end
end
```

Try refactoring this :D

Reply:
982491e70ce73171f97568eb422847be88c4bff2